incubator/fluentd-cloudwatch/templates/daemonset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
maybe use capabilities instead to make it dynamic so you can target older and newer apis with same chart: https://helm.sh/docs/chart_template_guide/builtin_objects/ ?